packages/headless/src/features/product-listing/product-listing-actions.ts

```diff
@@ -131,6 +135,12 @@ export const buildProductListingRequest = (
   };
 };
 
+function hasOneAdvancedParameterActive(
```

Comment:
Would `isDebugMode` be a clearer name for this function? If it will contain other things in the future, maybe we can rename then?